src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```diff
@@ -159,12 +161,13 @@ private void setCommonPreviewParams(SscsCaseData sscsCaseData, String endDate) {
 
     }
 
-    private void commonSetupForInterpreterDescription(SscsCaseData sscsCaseData) {
+    private NoticeIssuedTemplateBody setAdjournmentAndGetTemplateBody(SscsCaseData sscsCaseData, DynamicListItem item) {
         sscsCaseData.setAdjournCaseGenerateNotice("yes");
-        sscsCaseData.setAdjournCaseTypeOfNextHearing("faceToFace");
-        sscsCaseData.setAdjournCaseNextHearingDateType("firstAvailableDate");
-        sscsCaseData.setAdjournCaseTime(AdjournCaseTime.builder().build());
         sscsCaseData.setAdjournCaseInterpreterRequired("yes");
```

Comment:
You can also add adjournCaseGenerateNotice and adjournCaseInterpreterRequired to setup, I really don't think you need a method for this.

Reply:
Make sure to remove these from all the other tests as well, only overriding when they are different, e.g. 
![image](https://user-images.githubusercontent.com/18538046/199022614-b90c25c1-15a6-4bb2-93ff-fc809484ccd5.png)